### PR TITLE
Rewrite of the translations app from scratch

### PR DESCRIPTION
**Pull request by samaxes**

### CODE REVIEW

**Comment by jason-p-pickering** [comment]
@samaxes @MDHomem 
I am getting this error on invoking `yarn start`

```
jason@jason-XPS-13-9360:~/development/translations-app$ yarn start
yarn start v0.18.1
$ yarn manifest && yarn localize && node scripts/start.js 
yarn manifest v0.18.1
$ d2-manifest package.json ./public/manifest.webapp 
Reading package data: package.json
Validating manifest: ✓ Ok
Writing manifest to: ./public/manifest.webapp
Done!
Done in 0.14s.
yarn localize v0.18.1
$ d2-i18n-generate -n translations-app -p ./i18n/ -o ./src/locales/ 
sh: 1: d2-i18n-generate: not found
error Command failed with exit code 127.
error Command failed with exit code 1.
```

**Comment by jason-p-pickering** [comment]
@samaxes @MDHomem 
I am getting this error on invoking `yarn start`

```
jason@jason-XPS-13-9360:~/development/translations-app$ yarn start
yarn start v0.18.1
$ yarn manifest && yarn localize && node scripts/start.js 
yarn manifest v0.18.1
$ d2-manifest package.json ./public/manifest.webapp 
Reading package data: package.json
Validating manifest: ✓ Ok
Writing manifest to: ./public/manifest.webapp
Done!
Done in 0.14s.
yarn localize v0.18.1
$ d2-i18n-generate -n translations-app -p ./i18n/ -o ./src/locales/ 
sh: 1: d2-i18n-generate: not found
error Command failed with exit code 127.
error Command failed with exit code 1.
```

**Comment by jason-p-pickering** [comment]
@samaxes @MDHomem 
I am getting this error on invoking `yarn start`

```
jason@jason-XPS-13-9360:~/development/translations-app$ yarn start
yarn start v0.18.1
$ yarn manifest && yarn localize && node scripts/start.js 
yarn manifest v0.18.1
$ d2-manifest package.json ./public/manifest.webapp 
Reading package data: package.json
Validating manifest: ✓ Ok
Writing manifest to: ./public/manifest.webapp
Done!
Done in 0.14s.
yarn localize v0.18.1
$ d2-i18n-generate -n translations-app -p ./i18n/ -o ./src/locales/ 
sh: 1: d2-i18n-generate: not found
error Command failed with exit code 127.
error Command failed with exit code 1.
```

**Comment by jason-p-pickering** [comment]
@samaxes @MDHomem 
I am getting this error on invoking `yarn start`

```
jason@jason-XPS-13-9360:~/development/translations-app$ yarn start
yarn start v0.18.1
$ yarn manifest && yarn localize && node scripts/start.js 
yarn manifest v0.18.1
$ d2-manifest package.json ./public/manifest.webapp 
Reading package data: package.json
Validating manifest: ✓ Ok
Writing manifest to: ./public/manifest.webapp
Done!
Done in 0.14s.
yarn localize v0.18.1
$ d2-i18n-generate -n translations-app -p ./i18n/ -o ./src/locales/ 
sh: 1: d2-i18n-generate: not found
error Command failed with exit code 127.
error Command failed with exit code 1.
```

**Comment by jason-p-pickering** [comment]
@samaxes @MDHomem 
I am getting this error on invoking `yarn start`

```
jason@jason-XPS-13-9360:~/development/translations-app$ yarn start
yarn start v0.18.1
$ yarn manifest && yarn localize && node scripts/start.js 
yarn manifest v0.18.1
$ d2-manifest package.json ./public/manifest.webapp 
Reading package data: package.json
Validating manifest: ✓ Ok
Writing manifest to: ./public/manifest.webapp
Done!
Done in 0.14s.
yarn localize v0.18.1
$ d2-i18n-generate -n translations-app -p ./i18n/ -o ./src/locales/ 
sh: 1: d2-i18n-generate: not found
error Command failed with exit code 127.
error Command failed with exit code 1.
```

**Comment by jason-p-pickering** [comment]
@samaxes @MDHomem 
I am getting this error on invoking `yarn start`

```
jason@jason-XPS-13-9360:~/development/translations-app$ yarn start
yarn start v0.18.1
$ yarn manifest && yarn localize && node scripts/start.js 
yarn manifest v0.18.1
$ d2-manifest package.json ./public/manifest.webapp 
Reading package data: package.json
Validating manifest: ✓ Ok
Writing manifest to: ./public/manifest.webapp
Done!
Done in 0.14s.
yarn localize v0.18.1
$ d2-i18n-generate -n translations-app -p ./i18n/ -o ./src/locales/ 
sh: 1: d2-i18n-generate: not found
error Command failed with exit code 127.
error Command failed with exit code 1.
```

**Comment by jason-p-pickering** [comment]
@varl @larshelge I have reviewed the general functionality and think its good to go. Could you guys perform a review of the code  and see if there is anything which you would like to comment on?

**Comment by jason-p-pickering** [comment]
Other question maybe for @varl is whether we have packaged the Roboto font with the other apps? DHIS2 needs to be able to work potentially offline, so I think this also needs to be packaged within the app, or perhaps the commons resources app?

File: .gitignore [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+# production
+/build
+/target
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (varl) [inline .gitignore]: Could just be `.env.*.local` ?

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,27 @@
 language: node_js
 node_js:
-- '7'
+- '8'
 cache:
```
Comment (varl) [inline .travis.yml]: Remove `cache` property.

File: .env.template [hunk 3]
```diff
@@ -0,0 +1,3 @@
+REACT_APP_DHIS2_BASE_URL=http://localhost:8080
+REACT_APP_DHIS2_API_VERSION=30
```
Comment (varl) [inline .env.template]: If you have ejected and control, then config should be read from `DHIS2_HOME/config.json` like other apps for consistency.

File: public/css/material-icons.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/* fallback */
```
Comment (varl) [inline public/css/material-icons.css]: see general comment for external resources

File: public/css/roboto.css [hunk 5]
```diff
@@ -0,0 +1,336 @@
+/* cyrillic-ext */
```
Comment (varl) [inline public/css/roboto.css]: see general comment for external resources

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <title>Translations App</title>
+    <link type="text/css" rel="stylesheet" href="./css/material-icons.css">
+    <link type="text/css" rel="stylesheet" href="./css/roboto.css">
```
Comment (varl) [inline public/index.html]: see general comment for external resources

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+/* React */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* d2-ui */
+import D2UIApp from '@dhis2/d2-ui-app';
+import HeaderBar from '@dhis2/d2-ui-header-bar';
+
+/* components */
```
Comment (varl) [inline src/App.js]: Are pages components?

**Pull request merged**